hoptimator-operator/src/main/java/com/linkedin/hoptimator/operator/HoptimatorOperatorApp.java

```diff
@@ -37,10 +45,34 @@ public HoptimatorOperatorApp(String modelPath, String namespace, Properties prop
   }
 
   public static void main(String[] args) throws Exception {
-    if (args.length != 1) {
+    if (args.length<1) {
       throw new IllegalArgumentException("Missing model file argument.");
     }
-    new HoptimatorOperatorApp(args[0], "default", new Properties()).run();
```

Comment:
Some formatting issues here? Looks like 4 instead of 2 spaces.

Reply:
Are you referring to line 49 or 63-69?